mesonbuild/compilers/rust.py

```diff
@@ -55,6 +55,14 @@ class RustCompiler(Compiler):
         '3': ['-W', 'warnings'],
     }
 
+    mscrt_args = {
```

Comment:
Can we make this: `MSVCRT_ARGS: T.Mapping[str, T.List[str]] = ...`

All caps is PEP8 convention for "constant data", and making the type a `mapping` will let mypy yell at us if someone tries to mutate this. I would love to make this `ImmutableListProtocol`, but I think that may be an invasive change, and it's not in scope for this PR.

Reply:
done